core/src/main/java/bio/terra/pearl/core/service/kit/KitRequestDetails.java

```diff
@@ -0,0 +1,85 @@
+package bio.terra.pearl.core.service.kit;
+
+import bio.terra.pearl.core.model.kit.KitRequest;
+import bio.terra.pearl.core.model.kit.KitRequestStatus;
+import bio.terra.pearl.core.model.kit.KitType;
+import bio.terra.pearl.core.service.exception.internal.InternalServerException;
+import bio.terra.pearl.core.service.kit.pepper.PepperKit;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.node.ObjectNode;
+
+import java.time.Instant;
+import java.util.UUID;
+
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import lombok.experimental.SuperBuilder;
+import lombok.extern.slf4j.Slf4j;
+
+/**
+ * Holds set of kit request data that is relevant to users
+ */
+@Getter
+@Setter
+@SuperBuilder
+@NoArgsConstructor
+@Slf4j
+public class KitRequestDetails {
```

Comment:
I totally agree on `Dto`, I was about to comment that before I saw this. My only concern is that we use `Dto` in the populate side a lot, so it might be confusing, but it still communicates intent better.

Reply:
yeah, for clarity, we try to suffix the populate Dtos with  "PopDto". So either we could start calling these "UiDtos" or we just assume DTOs are for the UI unless otherwise named (I'd lean toward this).

Reply:
Wide open on naming. 'KitRequestDetails' was mostly a placeholder since I did not see a pattern like this in the code. Just tell me what you like.

Reply:
I'd vote for `KitRequestDto` personally, I agree that by default if I saw `Dto` I'd assume it was for serializing for the API/UI.